baseapp/baseapp_test.go

```diff
@@ -39,39 +39,31 @@ var (
 
 type paramStore struct {
 	db *dbm.MemDB
+	t  *testing.T
 }
 
 func (ps *paramStore) Set(_ sdk.Context, key []byte, value interface{}) {
 	bz, err := json.Marshal(value)
-	if err != nil {
-		panic(err)
-	}
-
-	ps.db.Set(key, bz)
+	require.NoError(ps.t, err)
+	err = ps.db.Set(key, bz)
+	require.NoError(ps.t, err)
```

Comment:
```suggestion
	require.NoError(ps.t, ps.db.Set(key, bz))
```

Reply:
There are 1000s of other lines in cosmos-sdk which you can be merged. I didn't want to merge it because this is how most of the code was written.

Reply:
Fair enough, it's a tiny suggestion. Feel free to ignore it.